config/config.go

```diff
@@ -55,6 +62,19 @@ type Module struct {
 	XXX map[string]interface{} `yaml:",inline"`
 }
 
+func (sc *SafeConfig) ReloadConfig(configFile string) (err error) {
```

Comment:
I think this should remain in the exporter code, as we'll be mixing in the generator config into this package at some point.